chia/_tests/core/consensus/stores/test_coin_store_protocol.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import annotations
+
+import pytest
+from chia_rs.sized_bytes import bytes32
+from chia_rs.sized_ints import uint32, uint64
+
+from chia._tests.util.db_connection import DBConnection
+from chia.consensus.coinbase import create_farmer_coin, create_pool_coin
+from chia.full_node.coin_store import CoinStore
+
+# black box tests from `chia/_tests/core/full_node/stores/test_coin_store.py`
+# should be moved here
+
+
+@pytest.fixture(scope="function", params=[2])
+def db_version(request: pytest.FixtureRequest) -> int:
+    return int(request.param)
+
+
```

Comment:
this seems to be a duplication of the existing fixture.  duplicating would defeat what i believe is it's purpose of having a single place to define a new db version to be tested against and then having all relevant tests parametried accordingly.

https://github.com/Chia-Network/chia-blockchain/blob/f782fa8341cf6caa2dc3dd37a4ee351905f92763/chia/_tests/conftest.py#L272-L274

```suggestion
```